### ISSUE TITLE
Default to Documents Directory

### ISSUE THREAD

**Issue by KPreston2900050** [stenzek/duckstation#850]
No matter what I do it refuses to stop defaulting to Documents.

I don't know about you but I don't make a habit of using my main system drive to save binary files or ISO's

Maybe that worked in the 1990's but times have changed defaulting to C:

The root directory of where the Dolphin Binary is a great place but if you do not agree then give people a CHOICE instead of forcing.

At this point I just deleted the whole thing off my computer even if it is better than most other PS1 emulators.

It just seems that in this day and age why would this even come up in a conversation when the norm is choice.

**Comment by backgamon** [comment]
Use "portable.txt".
After directory are the emu folder himself.

**Comment by stenzek** [comment]
Yeah, sorry, this really seems like a baseless rant which is not constructive.

DS uses the documents directory so that if you have different versions, it will share settings and memcards. It respects the path you configure - if you don't want documents in C drive, move it in windows.

As for saving it in the same directory as the program, did you not read the readme? It's right there. Specifically the last paragraph.
![Screenshot_20200915-093159](https://user-images.githubusercontent.com/11288319/93148229-9ba81500-f736-11ea-9a36-45c90eed8144.jpg)

Tldr; create portable.txt, or use -portable on the command line

**Comment by stenzek** [comment]
Closing as it's a non-issue - portable mode exists. Please read the readme next time before writing an angry issue! :)
